crates/swc_css_compat/src/utils.rs

```diff
@@ -0,0 +1,5 @@
+use swc_css_ast::{ComponentValue, Rule};
+
+pub(crate) fn rule_to_component_value(rule: Rule) -> ComponentValue {
```

Comment:
Is this required? If you need to use it with `.map`, you can do `.map(From::from)` intead.

Reply:
Just copy/paste from old code, yeah, let's fix it